test/simplifyExpression/basicsSearch/testSimplify.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const assert = require('assert');
+const math = require('mathjs');
+
+const flatten = require('../../../lib/util/flattenOperands');
+const print = require('../../../lib/util/print');
+
+
+function testSimplify(exprStr, outputStr, simplifyOperation) {
+	it(exprStr + ' -> ' + outputStr, function () {
+	    const inputNode = flatten(math.parse(exprStr));
+	    const newNode = simplifyOperation(inputNode).newNode;
+	    assert.equal(
+	      print(newNode),
+	      outputStr);
+	});
+}
+
+module.exports = testSimplify;
```

Comment:
really small nit, sorry, but can you go and add new lines at the bottom of the files that don't have them? (github shows you which ones they are with the :no_entry_sign: )